ROADMAP.md

```diff
@@ -82,52 +82,76 @@ Any dates listed below and the specific issues that will ship in a given milesto
   * Crossplane.io reflects the updated roadmap / vision [crossplaneio.github.io#22](https://github.com/crossplaneio/crossplaneio.github.io/issues/22)
-  * DevOps pipeline examples for Jenkins, GitLab, GitOps using Infra Stacks [#631](https://github.com/crossplaneio/crossplane/issues/631)
 
-## v0.4 - Template App Stacks & Infra Stacks Expansion
-* Stacks Manager
-  * Stack versioning, upgrade, & dependency resolution
+## v0.4.0 Stable v1beta1 APIs for use in CI/CD pipelines like GitLab
+* GitLab 12.5 Auto DevOps (ADO) integration phase 1 - provision managed PostgreSQL from GitLab ADO pipelines
+  * Subset of the overall [GitLab Auto DevOps integration](https://gitlab.com/groups/gitlab-org/-/epics/1866#note_216080986) 
+  * [Crossplane as a GitLab-managed app (phase1)](https://gitlab.com/gitlab-org/gitlab/issues/34702) - provision managed PostgreSQL from GitLab ADO pipelines
+  * Crossplane supporting work [#867](https://github.com/crossplaneio/crossplane/issues/867)
+
+* Stable v1beta1 Services APIs for managed databases and caches (GCP, AWS) [#863](https://github.com/crossplaneio/crossplane/issues/863)
+  * Align on shape of APIs & best practices
+    * Beta meta model w/ DB & Redis, so users can deploy to dev/test/prod
+    * Naming scheme for all resources.
+    * Managed resource name as external name for all resources.
+  * Upgrade GCP stack to v1beta1: CloudSQL and CloudMemoryInstance with high-def CRDs & controllers
+  * Upgrade AWS stack to v1beta1: RDS and ReplicationGroup with high-def CRDs & controllers
+
+* Cross-resource referencing for networks, subnets, and other resources [#707](https://github.com/crossplaneio/crossplane/issues/707) 
+  * Support `kubectl apply -f` for a directory of resources to cleanly support GitOps for both infrastructure and apps
+  * Sample infra and app repos you can `kubectl apply -f` and have a working environment quickly
+    * infrastructure (networks, subnets, managed k8s cluster, resource classes for databases, etc.)
+    * apps (e.g. kubernetes core resources for e.g. a Wordpress app plus the resource claims for managed service dependencies
+  * Update crossplane.io services guides and stacks guides to use `kubectl apply -f` technique
+
+* Claim-based provisioning of [Rook](https://rook.io/)-managed databases [#862](https://github.com/crossplaneio/crossplane/issues/862)
+  * Support for CockroachDB and Yugabyte DB
+
+ * Release automation for shorter release cycles and hot fixes [#864](https://github.com/crossplaneio/crossplane/issues/864) 
+   * Updating pipelines to include automation #6
+   * SonarCloud checks for cloud provider stacks #875
+   * crossplane-runtime build pipelines crossplaneio/crossplane-runtime#14
+
+ * Trace utility for enhanced debugging support. [#744] 
+
+ * Simple Resource Class Selection [#952]
+
+## Under consideration v0.5.0
```

Comment:
want to also add "default classes of service" that provide sane and reasonable defaults to remove initial setup friction and get started even quicker?

and is that related to example config repos / best practice repos?